gcc/testsuite/gcc.target/riscv/rvv/sf_vfnrclip.c

```diff
@@ -0,0 +1,21 @@
+#include "riscv_vector.h"
+
+vint8mf8_t test1(float vs1, vfloat32mf2_t vs2, size_t vl) {
+        return  __riscv_sf_vfnrclip_x_f_qf_i8mf8(vs2, vs1, vl);
+}
+
```

Comment:
Could you reference this file https://github.com/gcc-mirror/gcc/blob/master/gcc/testsuite/gcc.target/riscv/target-attr-01.c and add correct test directive?
e.g.
```c
/* { dg-do compile } */
/* { dg-skip-if "" { *-*-* } { "-flto" } { "" } } */
/* { dg-options "-march=rv64gcv_xsfvfnrclipxfqf -O2 -mabi=lp64d" } */
/* { dg-final { check-function-bodies "**" "" } } */
```

and add some check within the testcase like:
```c
/*
** foo:
**   ...
**   vsetvli\s*x0, a0, e32, m1, ta, ta
**   sf.vfnrclip.x.f.qf\s*fa0,v8
**   ...
*/
```

And don't forgot `sf_vqmacc.c`